db/migrate/20180924185921_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,7 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :provider, :string, :null => false, :default => "email"
```

Comment:
I know rubocop do not check these files, but better if you follow the convention even here:
`key: 'value'` and not `key: => 'value'`

Same for the ones below

Reply:
That migration was generated by the Devise gem but I can fix it.